daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md

```diff
@@ -905,24 +899,21 @@ dapr run --app-id batch-sdk --app-port 6002 --dapr-http-port 3502 --dapr-grpc-po
 The code inside the `process_batch` function is executed every 10 seconds (defined in [`binding-cron.yaml`]({{< ref "#componentsbinding-cronyaml-component-file" >}}) in the `components` directory). The binding trigger looks for a route called via HTTP POST in your Flask application by the Dapr sidecar.
 
 ```go
-func processCron(w http.ResponseWriter, r *http.Request) {
-	fileContent, err := os.Open("../../orders.json")
-}
+	// Triggered by Dapr input binding
+	r.HandleFunc("/"+cronBindingName, processBatch).Methods("POST")
 ```
 
 The `batch-sdk` service uses the PostgreSQL output binding defined in the [`binding-postgres.yaml`]({{< ref "#componentbinding-postgresyaml-component-file" >}}) component to insert the `OrderId`, `Customer`, and `Price` records into the `orders` table.
 
 ```go
-client, err := dapr.NewClient()
- // ...
-sqlCmd := fmt.Sprintf("insert into orders (orderid, customer, price) values (%d, '%s', %s);", order.OrderId, order.Customer, strconv.FormatFloat(order.Price, 'f', 2, 64))
-fmt.Println(sqlCmd)
-in := &dapr.InvokeBindingRequest{
-	Name:      bindingName,
-	Operation: "exec",
-	Data:      []byte(""),
-	Metadata:  map[string]string{"sql": sqlCmd},
-}
+	// Insert order using Dapr output binding via Dapr SDK
+	in := &dapr.InvokeBindingRequest{
+		Name:      sqlBindingName,
+		Operation: "exec",
+		Data:      []byte(""),
+		Metadata:  map[string]string{"sql": sqlCmd},
```

Comment:
Per above I think you might need to show dapr being declared as well as the sqlcmd.